assets/js/blocks/cart-checkout/checkout/edit.js

```diff
@@ -14,9 +9,7 @@ const Edit = ( { attributes } ) => {
 
 	return (
 		<div className={ className }>
-			<Disabled>
-				<Block attributes={ attributes } />
-			</Disabled>
+			<Block attributes={ attributes } />
```

Comment:
I removed them so that we can easily interact with the components within the editor, I think a @todo to remind us to put it back later should be fine?

Reply:
Why do you need to interact with them in the editor?  You could just have a tab that has them available on the frontend for testing interactions?